thirdparty/onnx/CMakeLists.txt

```diff
@@ -65,7 +65,7 @@ ov_install_static_lib(onnx_proto ${OV_CPACK_COMP_CORE})
 
 # WA for ONNX: protobuf must be in the same export set of ONNX targets
 # in case of protobuf::libprotobuf-lite / protobuf::libprotobuf are imported targets
-if(NOT ENABLE_SYSTEM_PROTOBUF)
+if(NOT ENABLE_SYSTEM_PROTOBUF AND NOT BUILD_SHARED_LIBS)
```

Comment:
Why do you enable it only for static build?

Reply:
it's installed only for cases when we built protobuf _manually_